src/core_plugins/kibana/public/management/_management_app.scss

```diff
@@ -0,0 +1,52 @@
+.mgtPanel {
+  margin-bottom: $euiSize;
+  background: $euiColorEmptyShade;
+}
+
+.mgtPanel__link {
+  @include euiFontSizeL;
+
+  &.mgtPanel__link--disabled {
+    opacity: $euiColorDarkShade;
+    cursor: default;
+
+    &:hover, &:visited {
+      color: $euiColorPrimary;
+    }
+  }
+}
+
+// SASSTODO: Remove when this form is replaced by EUI
+kbn-management-objects {
+  form {
+    margin-bottom: $euiSize;
+  }
+  .list-unstyled {
+    li {
+      border-bottom: $euiBorderThin;
+      padding: $euiSizeS;
+    }
+  }
+  .empty {
+    color: $euiColorDarkShade;
+  }
+
+  .item {
+    padding: $euiSizeM;
+
+    .item-title {
+      margin-left: $euiSizeL;
+    }
+
+    .actions {
+      margin-top: $euiSizeXS;
+    }
+  }
+
+  .header {
+    .title, .controls {
+      padding-right: 1em;
```

Comment:
Still a relative sizing, but for consistency should this be an EUI size variable?

Reply:
I'm leaving this one just to match against current Kibana syling for now. It'll all get replaced when the team comes in to replace this stuff.